webapp/src/js/enketo/medic-xpath-extensions.js

```diff
@@ -110,7 +110,7 @@ const asMoment = (r) => {
       const time = `${rDate}T00:00:00.000${getTimezoneOffsetAsTime(new Date(rDate))}`;
       return moment(time);
     }
-    return moment(r);
+    return rMoment;
```

Comment:
This saves us initialising the moment twice. It might be cleaner to return quickly but I think this is ok...